web/files/referencedby.go

```diff
@@ -10,6 +11,53 @@ import (
 	"github.com/labstack/echo/v4"
 )
 
+// ContentType is the official mime-type for JSON-API
+const ContentType = "application/vnd.api+json"
+
+// UpdatedReferences is a container for the updated list of referenced_by object
+// for an io.cozy.files document.
+type UpdatedReferences struct {
+	DocRev       string                 `json:"rev"`
+	ReferencedBy []couchdb.DocReference `json:"referencedBy"`
+}
+
+func NewUpdatedReferences(rev string, refs []couchdb.DocReference) UpdatedReferences {
+	u := UpdatedReferences{DocRev: rev}
+	u.ReferencedBy = make([]couchdb.DocReference, len(refs))
+	copy(u.ReferencedBy, refs)
+
+	return u
+}
+
+// MarshalUpdatedReferences serializes an UpdatedReferences to JSON.
+// It returns a json.RawMessage that can be returned in an HTTP response.
+func MarshalUpdatedReferences(r UpdatedReferences) (json.RawMessage, error) {
```

Comment:
I don't see what is the purpose of this function. I think that `data, err := json.Marshal(r)` is more straightforward than `data, err := MarshalUpdatedReferences(r)`.